frontend/src/Components/Form/SeriesTypeSelectInput.tsx

```diff
@@ -23,21 +23,21 @@ const seriesTypeOptions: ISeriesTypeOption[] = [
     key: seriesTypes.STANDARD,
     value: 'Standard',
     get format() {
-      return translate('StandardTypeFormat', { format: 'S01E05' });
+      return translate('StandardEpisodeTypeFormat', { format: 'S01E05' });
```

Comment:
These ones should also be unique to Sonarr.

Reply:
I changed these on purpose as I felt this looked better in en.json;

``` 
"StandardEpisodeFormat": "Standard Episode Format",
  "StandardEpisodeTypeDescription": "Episodes released with SxxEyy pattern",
  "StandardEpisodeTypeFormat": "Season and episode numbers ({format})",
```

instead of
``` 
"StandardEpisodeFormat": "Standard Episode Format",
  "StandardTypeDescription": "Episodes released with SxxEyy pattern",
  "StandardTypeFormat": "Season and episode numbers ({format})",
```


let me know if I should revert this for standard, anime and daily?